api/endpoints/wmts.py

```diff
@@ -161,12 +174,15 @@ def get(self, z, x, y, ext, color_map=None, rescale=None):
 
 
 def get_mosaic_tilejson(urls_query_string=''):
-    mosaic_tilejson_url = settings.TILER_ENDPOINT + '/mosaic/tilejson.json?urls=' + urls_query_string
+    #mosaic_tilejson_url = settings.TILER_ENDPOINT + '/mosaic/tilejson.json?urls=' + urls_query_string
```

Comment:
Remove commented code in this file, thanks!

Reply:
And elsewhere in this PR!

Reply:
Fixed! Thanks!